docs/experiments-msmarco-v2-tct_colbert-v2.md

```diff
@@ -8,20 +8,32 @@ The model is described in the following paper:
 At present, all indexes are referenced as absolute paths on our Waterloo machine `orca`, so these results are not broadly reproducible.
 We are working on figuring out ways to distribute the indexes.
 
-For the TREC 2021 Deep Learning Track, we applied our TCT-ColBERTv2 model trained on MS MARCO (V1) in a zero-shot manner.
-Specifically, we applied inference over the MS MARCO V2 [passage corpus](https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-v2.md#passage-collection) and [segmented document corpus](https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-v2.md#document-collection-segmented) to obtain the dense vectors.
+For the TREC 2021 Deep Learning Track, we tried two different approaches:
 
-Let's prepare our environment variables:
+1. We applied our TCT-ColBERTv2 model trained on MS MARCO (V1) in a zero-shot manner.
+2. We retrained our TCT-ColBERTv2 model on the MS MARCO (V2) passage data.
+
+In both cases, we applied inference over the MS MARCO V2 [passage corpus](https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-v2.md#passage-collection) and [segmented document corpus](https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-v2.md#document-collection-segmented) to obtain the dense vectors.
+
+These are indexes and encoder for the zero-shot (V1) models:
 
 ```bash
 export PASSAGE_INDEX0="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.0shot.msmarco-passage-v2-augmented"
 export DOC_INDEX0="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.0shot.msmarco-doc-v2-segmented"
 export ENCODER0="castorini/tct_colbert-v2-hnp-msmarco"
 ```
 
-## Passage V2
+These are indexes and encoder for the retrained (V2) models:
 
-Dense retrieval with TCT-ColBERT-V2, brute-force index:
+```bash
+export PASSAGE_INDEX1="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.psg_v2_ft.msmarco-passage-v2-augmented"
+export DOC_INDEX1="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.psg_v2_ft.msmarco-doc-v2-segmented"
+export ENCODER1="/store/scratch/indexes/trec2021/torch_ckpt/tct_colbert-v2-hnp-msmarco-hn-msmarcov2"
```

Comment:
We've uploaded the checkpoint to hgf: [modelcard](https://huggingface.co/castorini/tct_colbert-v2-hnp-msmarco-r2).
Maybe we can revise ENCODER1 as `castorini/tct_colbert-v2-hnp-msmarco-r2`?

Reply:
Yup, will do!